frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -32,13 +40,21 @@ type ReviewRow = {
 
 const AdminDashboard = (): React.ReactElement => {
   const [data, setData] = useState<ReviewResponse[]>([]);
+  const [isOpen, setIsOpen] = useState(false);
 
   useEffect(() => {
     reviewAPIClient.getReviews().then((allReviews: ReviewResponse[]) => {
       setData(allReviews);
     });
   }, []);
 
+  const deleteReview = async (reviewId: number, reviewIndex: number) => {
+    await reviewAPIClient.deleteReviewById(reviewId.toString());
+    const newData = [...data];
+    newData.splice(reviewIndex, 1);
```

Comment:
(minor) Maybe we can combine these into 1 line, ie `const newData = [...newData.splice(reviewIndex, 1)];`